lib/gli.rb

```diff
@@ -24,10 +24,20 @@
 
 module GLI
   include GLI::App
+  @@use_hyphenized_form = false
```

Comment:
This isn't where this stuff should go.  Look at `app.rb` for lots of examples.  I think `hide_commands_without_desc` is a good template to follow for this.

The name should also be pretty specific as to its purpose.  Maybe `dasherize_underscore_options`?  Taking from Rails' [`dasherize`](https://github.com/rails/rails/blob/861b70e92f4a1fc0e465ffcf2ee62680519c8f6f/activesupport/lib/active_support/inflector/methods.rb#L186)